src/main/kotlin/com/bridgecrew/ui/vulnerabilitiesTree/CheckovFileTreeNode.kt

```diff
@@ -0,0 +1,37 @@
+package com.bridgecrew.ui.vulnerabilitiesTree
+
+import com.intellij.icons.AllIcons
```

Comment:
from my web experience, importing a lot of icons is intensive and the best practice is to import only what you use

Reply:
We're not actually importing any icon until we're using it, so there's no benefit in importing them explicitly. For example, the `AllIcons.Nodes.Folder` under the hood calls `load("nodes/folder.svg", 7796752220391063365L, 0);`